scipy/stats/stats.py

```diff
@@ -3116,6 +3117,17 @@ def trim_mean(a, proportiontocut, axis=0):
 F_onewayResult = namedtuple('F_onewayResult', ('statistic', 'pvalue'))
 
 
+class F_onewayConstantInputWarning(RuntimeWarning):
+    """Warning generated by `f_oneway` when an input is constant, e.g.
+       each of the samples provided is a constant array"""
+
+    def __init__(self, msg=None):
+        if msg is None:
+            msg = ("Each of the input arrays is constant;"
+                    "the F-value and p-value are not defined or infinite")
```

Comment:
suggest to change F-value to F-statistic

Reply:
Shouldn't we also change to ``F-statistic`` here? "F-value" is in documentation now, and I think the change must be kind of consistent
https://github.com/scipy/scipy/blob/b818978f3580e05a3885006cae337f28bd2aaf7d/scipy/stats/stats.py#L3140